plugin/evm/validators_manager_test.go

```diff
@@ -0,0 +1,65 @@
+package evm
+
+import (
+	"encoding/binary"
+	"math/rand"
+	"testing"
+
+	"github.com/flare-foundation/flare/ids"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func randArray() [20]byte {
+	data := [20]byte{}
+	rand.Read(data[:])
+
+	return data
+}
```

Comment:
```suggestion
func randShortID() ids.ShortID {
	id := ids.ShortID{}
	rand.Read(id[:])

	return id
}
```

Reply:
OKay I fixed this https://github.com/flare-foundation/coreth/pull/21/commits/b5d59c2475c2f081d8ccb493da8daedbd87067ff#diff-53d507a9b6fbf6d169f0b9686ad727450a293914f998f4a17c524fa84623c4d7R12
but now I see I made it randID(), should also work I think, yes?

Reply:
It's fine yes :)